utilities/backupable/backupable_db.cc

```diff
@@ -1407,16 +1407,19 @@ Status BackupEngineImpl::InsertPathnameToSizeBytes(
     std::unordered_map<std::string, uint64_t>* result) {
   assert(result != nullptr);
   std::vector<Env::FileAttributes> files_attrs;
-  Status status = env->GetChildrenFileAttributes(dir, &files_attrs);
-  if (!status.ok()) {
-    return status;
+  Status status = env->FileExists(dir);
+  if (status.ok()) {
+    status = env->GetChildrenFileAttributes(dir, &files_attrs);
```

Comment:
What will happen if we call this function and the directory doesn't exist? Although I'm OK with the approach now, if we can avoid to add this extra HDFS namenode call, it will be better. Every HDFS namenode call is expensive and slow.